models/DataObject/ClassDefinition.php

```diff
@@ -1329,20 +1314,7 @@ public function getLinkGenerator()
     /**
      * @return bool
      */
-    public function getCacheRawRelationData(): bool
-    {
-        return $this->cacheRawRelationData;
-    }
-
-    /**
-     * @param bool $cacheRawRelationData
-     *
-     * @return $this
-     */
-    public function setCacheRawRelationData($cacheRawRelationData)
-    {
-        $this->cacheRawRelationData = (bool) $cacheRawRelationData;
-
-        return $this;
+    public function getCacheRawRelationData() {
```

Comment:
You want to keep that for compatibility reasons? 
As this was just used internally I'd just kick it.